OMCompiler/Compiler/BackEnd/BackendDump.mo

```diff
@@ -830,9 +817,9 @@ algorithm
   print("dependents [or outputs] (" + intString(listLength(diffedVars)) + ")\n");
```

Comment:
Maybe start dependents with capital case

Reply:
Updated it